tests/test_image.py

```diff
@@ -14,3 +16,41 @@ def test_from_url_with_non_OK_response(httpserver: HTTPServer):
 
     with raises(RequestException) as e:
         Image.from_url(httpserver.url_for(path))
+
+
+def test_from_image_source_on_local_file_path_pathlib():
+    image_source_path = Path(__file__).parent / "dog-and-cat-cover.jpg"
+    img_1 = ImagePrompt.from_image_source(image_source=image_source_path)
```

Comment:
We should use the _new_ name `Image` in the tests

Reply:
Replaced all `ImagePath`s